Index one snapshot from upcoming lm10 release in `dev` (#7617) (#7648)
DataBiosphere/azul: UPGRADING.rst

have too many entries in this file.

#7617 Index lm10 in LungMAP for dev
===================================

Using the sandbox deployment's ``environment.py`` as a model, add the new
datasource in the ``lm2`` catalogs to your personal deployment's

> ``environment.py``.

Reviewer: Should also cover removal of `lungmap` catalog.

have too many entries in this file.

#7617 Index one snapshot from upcoming lm10 release in `dev`
============================================================

> In your HCA personal deployments' ``environment.py`` files:

Reviewer: ```suggestion
In your personal HCA deployments:
```

> - Make the ``lm2`` catalog the only active LungMAP catalog

Reviewer: ```suggestion
- Remove the ``lungmap`` catalog
```

> - Add the new datasource in the ``lm2`` catalog

Reviewer: ```suggestion
- Add the snapshot for project ``fdadee7e…`` to the ``lm2`` catalog
```
